api/src/opentrons/protocols/runner/json_proto/command_translator.py

```diff
@@ -1,18 +1,16 @@
-from typing import Dict, Iterable
+from typing import Dict, List
 
-from opentrons.protocol_engine import WellLocation, WellOrigin
-from opentrons.protocol_engine.commands import (
-    CommandRequestType, AspirateRequest, DispenseRequest,
-    PickUpTipRequest, DropTipRequest
-)
-from opentrons.protocols.models import json_protocol as models
+import opentrons.types
```

Comment:
I find `import foo.bar` to be very confusing; it's was non-obvious to me that you then proceed to use `foo.bar` in the code.

Given that we use exactly two structures from here, I would've written this as:

```suggestion
from opentrons.types import DeckSlotName, MountType
```

Mostly because I'm lazy and it's less typing everywhere, and doesn't feel to me like it's a loss in understanding because the name is consistent everywhere and most of us are working in editors / IDEs with type info + click-to-follow